converters/balanceToFloat.go

```diff
@@ -43,7 +45,7 @@ func (bc *balanceConverter) ComputeESDTBalanceAsFloat(balance *big.Int) float64
 }
 
 func (bc *balanceConverter) computeBalanceAsFloat(balance *big.Int, balancePrecision float64) float64 {
-	if balance == nil || balance == big.NewInt(0) {
+	if balance == nil || balance == zero {
```

Comment:
this big int comparison won't work. please compare the bytes of the big int values